locales/en.json

```diff
@@ -127,27 +129,32 @@
     "delete_list": {
       "cancel": "Cancel",
       "confirm": "Delete",
-      "title": "Are you sure you want to delete the \"{0}\" list?"
+      "description": "Are you sure you want to delete the \"{0}\" list?",
+      "title": "Delete list"
     },
     "delete_posts": {
       "cancel": "Cancel",
       "confirm": "Delete",
-      "title": "Are you sure you want to delete this post?"
+      "description": "Are you sure you want to delete this post?",
+      "title": "Delete post"
     },
     "mute_account": {
       "cancel": "Cancel",
       "confirm": "Mute",
-      "title": "Are you sure you want to mute {0}？"
+      "description": "Are you sure you want to mute {0}?",
+      "title": "Mute account"
     },
     "show_reblogs": {
       "cancel": "Cancel",
       "confirm": "Show",
-      "title": "Are you sure you want to show boosts from {0}？"
+      "description": "Are you sure you want to show boosts from {0}?",
+      "title": "Show boosts"
     },
     "unfollow": {
       "cancel": "Cancel",
       "confirm": "Unfollow",
-      "title": "Are you sure you want to unfollow?"
+      "description": "Are you sure you want to unfollow {0}?",
```

Comment:
As explained in the above comment, I added `{0}` for `en.json` but couldn't add other languages. Is it OK to keep other messages without `{0}`?

Reply:
Can you add a new one entry and use `te` to check if present?

Reply:
If missing new one use current otherwise use the new one with `{0}`

Reply:
IIRC `te` is for checking entry.

Reply:
upps, forgot it, we'll need to add the title, I'll add the title for spanish and try to add focus trap

Reply:
rn all confirm dialog titles will be using `en`  entry